webroot/admin/user-mgmt.php

```diff
@@ -42,9 +43,17 @@
         echo "<td>" . $user->getFirstname() . " " . $user->getLastname() . "</td>";
         echo "<td>" . $user->getUID() . "</td>";
         echo "<td>" . $user->getOrg() . "</td>";
-        echo "<td><a href='mailto:" . $user->getMail() . "'>" . $user->getMail() . "</a></td>";
+        echo "<td><a href ='mailto:" . $user->getMail() . "'>" . $user->getMail() . "</a></td>";
+        echo "<td>";
+        $cur_user_groups = $user->getGroups();
+        foreach($cur_user_groups as $cur_group) {
+            echo "<a href='mailto:" . $cur_group->getOwner()->getMail() . "'>" . $cur_group->getPIUID() . "</a>";
+            if ($cur_group !== array_key_last($cur_user_groups)) {
+                echo '<br>';
+        }
```

Comment:
```suggestion
            }
```
Really small change, just make sure these brackets line up with each other